src/app/shell/admin-tools/data/history-log/history-log.component.ts

```diff
@@ -52,21 +42,24 @@ export class HistoryLogComponent implements OnInit, OnDestroy {
   providerAdminHistory$: Observable<SearchResponse<ProviderAdminHistory[]>>;
   @Select(AdminState.applicationHistory)
   applicationHistory$: Observable<SearchResponse<ApplicationHistory[]>>;
-  @Select(PaginatorState.tableItemsPerPage)
-  tableItemsPerPage$: Observable<number>;
 
   destroy$: Subject<boolean> = new Subject<boolean>();
   tabIndex = 0;
   searchString: string;
   currentPage: PaginationElement = PaginationConstants.firstPage;
   searchFormControl = new FormControl('');
   dropdownData: DropdownData[];
-  filters: FilterData;
+  filters: FilterData = {
```

Comment:
I think this is bad naming (takin in the account that previously it was filters indeed, but now it includes pagination params), so let's remain it at least in filterParams